Lesson 08/todoist/models/task.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+// Task schema.
+const taskSchema = new mongoose.Schema({
+	userId: {type: mongoose.Schema.Types.ObjectId, required: true},
```

Comment:
I'd recommend u using "ref" attribute to add a reference to another collection. It will help u with fields population (kind of join in MongoDB). Further reading - http://mongoosejs.com/docs/populate.html

Reply:
Got it. Thanks for the link.